src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1526,12 +1526,21 @@ k
         cc.save();
         cc.translate(this.canvas.panX, this.canvas.panY);
 
+        // TODO: Remove the jQuery in favor of _each()!
         $.each(airport.terrain || [], (elevation, terrainLevel) => {
+            if (elevation < 1000) {
+                console.warn(`${airport.icao}.geojson contains 'terrain' at or` +
+                    ' below sea level, which is not supported!');
+
+                // within `$.each()`, this return acts like `continue;`
+                return;
+            }
+
             max_elevation = Math.max(max_elevation, elevation);
-            const color = `rgba(${UiController.terrain.COLOR[elevation]}, `;
+            const color = `hsla(${this.theme.TERRAIN.COLOR[elevation]}`;
 
-            cc.strokeStyle = `${color} ${UiController.terrain.BORDER_OPACITY})`;
-            cc.fillStyle = `${color} ${UiController.terrain.FILL_OPACITY})`;
+            cc.strokeStyle = `${color}, ${this.theme.TERRAIN.BORDER_OPACITY})`;
```

Comment:
🤔 

Hmm, I guess I thought this was in a few more places.  my bad.  I guess we can leave it as-is then.  I'll allow it 😉